vllm/model_executor/models/__init__.py

```diff
@@ -15,6 +15,7 @@
     "BaiChuanForCausalLM": ("baichuan", "BaiChuanForCausalLM"),  # baichuan-7b
     "BaichuanForCausalLM": ("baichuan", "BaichuanForCausalLM"),  # baichuan-13b
     "BloomForCausalLM": ("bloom", "BloomForCausalLM"),
+    "ChameleonForCausalLM": ("chameleon", "ChameleonForConditionalGeneration"),
```

Comment:
This was left unmatched on purpose since HF hasn't fixed the model repo. 

See  https://github.com/huggingface/transformers/issues/32098

Reply:
Add a comment/todo in the file?